api/src/server/websocket.rs

```diff
@@ -174,25 +174,31 @@ impl Actor for ProgramWebsocket {
                 ctx.stop();
             } else {
                 // Not timed out, send another ping
-                ctx.ping("");
+                ctx.ping(b"");
```

Comment:
this b string makes me mad for no reason lmao

Reply:
triggered by python?